metron-platform/metron-elasticsearch/metron-elasticsearch-storm/pom.xml

```diff
@@ -36,6 +36,11 @@
             <artifactId>elasticsearch-shaded</artifactId>
             <version>${project.parent.version}</version>
         </dependency>
+        <dependency>
```

Comment:
What's going on here - were we sneaking in the ES client dep via a transitive dep from the integration tests?

Reply:
Nevermind - I dove into the code before reading your comments ;)